SERVER/db/tables.js

```diff
@@ -0,0 +1,91 @@
+import {Pool}from "pg";
+import dotenv from "dotenv";
+dotenv.config();
+
+const pool=new Pool({
+    connectionString:process.env.DATABASE_URL
+});
+pool.on('connect',()=>{
```

Comment:
Why not import this pool connection from your `db.js` file you already created.

Reply:
done